e2e/utils/visual-regression.js

```diff
@@ -3,9 +3,30 @@ function getMisMatchPercentage(results) {
 }
 
 module.exports.takeScreenshotAndGetWholePageCompareResult = (misMatchTolerance) => {
-  return ((getMisMatchPercentage(browser.checkViewport()) <= misMatchTolerance) || false);
+  const misMatchPercentage = getMisMatchPercentage(browser.checkViewport());
+  const isTestPassed = (misMatchPercentage <= misMatchTolerance) || false;
+  if (isTestPassed) {
+    return isTestPassed;
+  }
+  console.log('misMatchTolerance: ' + misMatchTolerance
+  + '\nmisMatchPercentage: ' + misMatchPercentage);
+
+  return isTestPassed;
 };
 
-module.exports.takeScreenShotOfElement = (elementSelector, misMatchTolerance) => {
-  return ((getMisMatchPercentage(browser.checkElement(elementSelector)) <= misMatchTolerance) || false);
+module.exports.takeScreenShotOfElement = (elementSelector, misMatchTolerance, ignoreComparison) => {
+  let ignoreComparisonValue = '';
+  if (ignoreComparison === true) {
+    ignoreComparisonValue = 'colors';
+  }
+  const misMatchPercentage = getMisMatchPercentage(browser.checkElement(elementSelector, { ignoreComparison: ignoreComparisonValue }));
+  const isTestPassed = (misMatchPercentage <= misMatchTolerance) || false;
+  if (isTestPassed) {
+    return isTestPassed;
+  }
+  console.log('misMatchTolerance: ' + misMatchTolerance
```

Comment:
Your solution here is totally fine. 👍 

I'm just letting you know that you can ease your life a bit with the so-called [template-literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals):

You can write this `console.log()` without concatenation like this and have the same output:

```js
console.log(`mismatchTolearance: ${misMatchTolerance}
misMatchPercentage: ${misMatchPercentage}
problematic elementSelector: ${elementSelector}`);
```